src/main/java/se/kth/sda/skeleton/comments/Comment.java

```diff
@@ -1,9 +1,15 @@
 package se.kth.sda.skeleton.comments;
 
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
+import com.fasterxml.jackson.annotation.*;
+import javax.persistence.*;
+import javax.validation.constraints.*;
+import se.kth.sda.skeleton.posts.Post;
+
+/**
+ * Represents a comment on a {@link Post} as a JPA Entity. This implementation of comment will autogenerate a primary key of type
+ * {@link Long} to indicate the current Comment entity. A Comment has a non-null field called {@code relatedPost}. The {@code relatedPost}
+ * is a {@link Post} to which the Comment is related. The body of a {@code Comment} cannot be null and is limited to 1500 characters.
+ */
 
```

Comment:
remove empty line between end of javadoc and start of class declaration

Reply:
done